cprt_bash_aliases

```diff
@@ -0,0 +1,8 @@
+#!/bin/bash
+
+# CPRT bash aliases
+# firstTimeInstall.sh will handle adding code to bashrc to source this file
+
+cprt_bash_alias_dir=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+export CPRT_BASH_ALIAS_FILE="$cprt_bash_alias_dir/cprt_bash_aliases"
+alias eaCPRT='vim $CPRT_BASH_ALIAS_FILE'
```

Comment:
'vim $CPRT_BASH_ALIAS_FILE && source ~/.bashrc && echo "Aliases Updated''

This is untested but something along these lines makes it more user friendly.

Reply:
Fixed and tested. Also I renamed the env variable so that it doesn't start with CPRT to avoid it cluttering auto completes for when use the terminal.